### PR TITLE
Translate setup/cri.md in Korean

### LINKED ISSUE #12166: Translate setup/cri in Korean
**This is a...** 

- [x] Feature Request
- [ ] Bug Report

**Problem:**
No setup/cri.md in Korean

**Proposed Solution:**
Translate setup/cri.md in Korean

**Page to Update:**
https://kubernetes.io/ko/docs/setup/cri

/language ko

### PR DESCRIPTION
**Pull request by seokho-son**
Translate setup/cri.md in Korean. To Fix #12166.

### CODE REVIEW

File: content/ko/docs/setup/cri.md [hunk 1]
```diff
@@ -0,0 +1,229 @@
...
+{{% capture body %}}
+
+다음의 커맨드들은 사용자의 운영체제에 따라 root로서 실행하길 바란다.
+각 호스트에 SSH 접속 후 `sudo -i` 실행을 통해서 root 사용자가 될 수 있을 것이다.
+
+## Docker
+
+각 머신들에 대해서, Docker를 설치한다.
+버전 18.06이 추천된다. 그러나 1.11, 1.12, 1.13, 그리고 17.03도 동작하는 것으로 알려져 있다. 
+쿠버네티스 릴리즈 노트를 통해서, 최신에 검증된 Docker 버전의 지속적인 파악이 필요하다.
```
Comment (gochist) [inline content/ko/docs/setup/cri.md]: s/릴리즈/릴리스/

File: content/ko/docs/setup/cri.md [hunk 2]
```diff
@@ -0,0 +1,229 @@
...
+apt-get install -y libseccomp2
+{{< /tab >}}
+{{< tab name="CentOS/RHEL 7.4+" codelang="bash" >}}
+yum install -y libseccomp
+{{< /tab >}}
+{{< /tabs >}}
+
+### Containerd 설치
+
+[Containerd 릴리즈](https://github.com/containerd/containerd/releases)는 주기적으로 출판된다. 아래의 값들은 작성 당시에 가용한 최신 버전을 기준으로 하드코드 되었다. 새로운 버전과 해시는 [여기](https://storage.googleapis.com/cri-containerd-release)에서 참고한다.
```
Comment (gochist) [inline content/ko/docs/setup/cri.md]: s/릴리즈/릴리스/

Comment (seokho-son) [reply content/ko/docs/setup/cri.md]: @gochist 리뷰 적용하였습니다. 감사합니다!

**Comment by gochist** [comment]
@seokho-son 감사합니다!
/lgtm
/approve

**Pull request merged**